### ISSUE TITLE
[suggestions] Online Trade with other players and Shop UI Improvements

### ISSUE THREAD

**Issue by UltraUnitilezer9909** [PavelZinchenko/event-horizon-frontier-builds#82]
Suggestions:
- Make the player gain credits or star selling item/module to other players and can be unlock mid game to prevent players get late game module during early game, this include private or code-only trade or public trade for any random trader to trade with...
- Make Bazaar (shop arena), Merchant and Smuggler's UI a bit more "impact" so more things can be viewed in a single screen
- Selling modules on Shops is a pain when you have thousands of modules each item and buff, instead of that, make all module have each folder and inside is there buffs and rarity...

thats all my suggestions, thank you for reading :D

**Comment by Finront** [comment]
I agree with the more organized merchants and organized models but trading may caused problems like scamming and maybe less challenging game play and possibly others too

**Comment by UltraUnitilezer9909** [comment]
> I agree with the more organized merchants and organized models but trading may caused problems like scamming and maybe less challenging game play and possibly others too

Let Pavel thing about it.... Besides it's not a "must have" suggestion...
